### DISCUSSION TITLE
RSSI information

### DISCUSSION THREAD

**Discussion by Audiobuzz** [jvde-github/AIS-catcher#30] [Ideas]
Is there any way of extracting RSSI information? either per message or some average statistics? This can be useful for tuning gain settings or testing different antennas. I also run an ads-b antenna and plot the rssi statistics over time to see what affect antenna changes make so this would be useful on AIS as well.

Many thanks

AB

**Comment by jvde-github** [comment]
Thanks for raising. If you run AIS-catcher with the switch ``-M D`` it will report out something which I have called ``signalpower``, It is not 100% the same as RSSI but it gives you some indication of the level of the signal.

It was discussed [here](https://github.com/jvde-github/AIS-catcher/issues/23) and called ``lvl`` at the time but since I have renamed it to ``signalpower``. 

Is this useful for you?

To process this you might want to run with ``-o 3`` so that the messages and meta-data like the signalpower are displayed as json.
